src/CoreShop/Bundle/CoreBundle/Resources/public/pimcore/js/report/abstract.js

```diff
@@ -88,18 +92,56 @@ coreshop.report.abstract = Class.create(pimcore.report.abstract, {
             if (grid) {
                 this.panel.add(grid);
             }
-
-            this.filter();
         }
 
         return this.panel;
     },
 
     getFilterFields: function () {
+
+        var _ = this,
+            store = pimcore.globalmanager.get('coreshop_stores');
+
         return [
+            {
```

Comment:
Does every report has a Store Filter?

Reply:
yes

Reply:
do future reports have a store filter as well? I am only concerned about limiting ourself in some way, maybe we can create a abstractStoreReport class?